rse-bash.Rmd

```diff
@@ -913,339 +634,538 @@ draft.txt
+
+     -w      The number of words in each input file is written to the standard
+             output.
+
+     When an option is specified, wc only reports the information requested by
+     that option.  The order of output always takes the form of line, word,
+     byte, and file name.  The default action is equivalent to specifying the
+     -c, -l and -w options.
+
+     If no files are specified, the standard input is used and no file name is
+     displayed.  The prompt will accept input until receiving EOF, or [^D] in
+     most environments.
+
+ENVIRONMENT
+     The LANG, LC_ALL and LC_CTYPE environment variables affect the execution
+     of wc as described in environ(7).
+
+EXIT STATUS
+     The wc utility exits 0 on success, and >0 if an error occurs.
+
+EXAMPLES
+     Count the number of characters, words and lines in each of the files
+     report1 and report2 as well as the totals for both:
+
+           wc -mlw report1 report2
+
+COMPATIBILITY
+     Historically, the wc utility was documented to define a word as a "maxi-
+     mal string of characters delimited by <space>, <tab> or <newline> charac-
+     ters".  The implementation, however, did not handle non-printing charac-
+     ters correctly so that "  ^D^E  " counted as 6 spaces, while
+     "foo^D^Ebar" counted as 8 characters.  4BSD systems after 4.3BSD modi-
+     fied the implementation to be consistent with the documentation.  This
+     implementation defines a "word" in terms of the iswspace(3) function,
+     as required by IEEE Std 1003.2 ("POSIX.2").
+
+SEE ALSO
+     iswspace(3)
+
+STANDARDS
+     The wc utility conforms to IEEE Std 1003.1-2001 ("POSIX.1").
+
+HISTORY
+     A wc command appeared in Version 1 AT&T UNIX.
+
+BSD                            February 23, 2005                           BSD
+```
+
+> **Navigating the Manual**
 >
-> If you make this kind of mistake, you can escape out of this state by holding down
-> the control key (<kbd>Ctrl</kbd>) and typing the letter <kbd>C</kbd>
-> once and letting go of the <kbd>Ctrl</kbd> key.
-> <kbd>Ctrl</kbd>+<kbd>C</kbd>
+> If our screen is too small to display an entire manual page at once,
+> the shell will use a [pager][pager] called `less` to show it piece by piece.
+> We can use <kbd>↑</kbd> and <kbd>↓</kbd> to move line-by-line
+> or <kbd>Ctrl</kbd>+<kbd>Spacebar</kbd> and <kbd>Spacebar</kbd>
+> to skip up and down one page at a time.
+> (<kbd>B</kbd> and <kbd>F</kbd> also work.)
+> 
+> To search for a character or word,
+> use <kbd>/</kbd> followed by the character or word to search for.
+> If the search produces multiple hits,
+> we can move between them using <kbd>N</kbd> (for "next").
+> To quit, press <kbd>Q</kbd>.
+
+This is a lot of information,
+most of which isn't really useful.
+Some commands have a `--help` option to provide a succinct summary of possibilites,
+but the best place to go for help these days is probably the [TLDR][tldr] website.
```

Comment:
There is also a tldr shell program one can use instead/in addition to of man pages.